### ISSUE TITLE
Grey out keys that aren't ENABLED and thus unavailable to use?

### ISSUE THREAD

**Issue by mxmilkiib** [remap-keys/remap#803]
Can these keys' usability status be detected via WebHID?

**Comment by yoichiro** [comment]
@mxmilkiib I would like to know more detail to understand the issue you have. What is the "Grey out keys that aren't ENABLED"?

**Comment by mxmilkiib** [comment]
I don't think this is possible. As in when a key is removed from a board making what is available less than the stored layout that Remap knows. I don't think boards can report this kind of information though(?)
